pkg/network/vmispec/hotplug_test.go

```diff
@@ -22,33 +22,38 @@ package vmispec_test
 import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
 	v1 "kubevirt.io/api/core/v1"
 
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	libvmistatus "kubevirt.io/kubevirt/pkg/libvmi/status"
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 )
 
-const (
-	extraNetworkName           = "othernet"
-	extraNetworkAttachmentName = "othernad"
-)
-
 var _ = Describe("utilitary funcs to identify attachments to hotplug", func() {
 	const (
 		guestIfaceName = "eno123"
 		nadName        = "nad1"
 		networkName    = "n1"
 	)
-
-	DescribeTable("NetworksToHotplugWhosePodIfacesAreReady", func(vmi *v1.VirtualMachineInstance, networksToHotplug ...v1.Network) {
+	DescribeTable("NetworksToHotplugWhosePodIfacesAreReady", func(vmiOpts []libvmi.Option, networksToHotplug ...v1.Network) {
+		vmi := libvmi.New(vmiOpts...)
```

Comment:
I would prefer you pass in the VMI because it looks nicer (when we read it) than passing libvmi options.
A few more characters will not make a lot of difference. If I think about it, it will be even shorter as you replace `[]libvmi.Option{}` with `libvmi.New()`.

E.g.:
```go
Entry("VMI without networks in spec does not have anything to hotplug", libvmi.New()),
```